src/scripts/index.js

```diff
@@ -0,0 +1,336 @@
+import "../pages/index.css";
+import Api from "../utils/Api.js";
+import {
+  checkFormValidity,
+  enableValidation,
+  disableButton,
+} from "./validation.js";
+
+// Api
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "6a49bc73-4cbf-41b7-bd3f-9c47b1919869",
+    "Content-Type": "application/json",
+  },
+});
+
+// Profile
+const profile = document.querySelector(".profile");
+const avatar = profile.querySelector(".profile__avatar");
+const name = profile.querySelector(".profile__name");
+const about = profile.querySelector(".profile__about");
+const editBtn = profile.querySelector("#profile-btn");
+const avatarBtn = profile.querySelector("#avatar-btn");
+
+function setData(data) {
+  setInfo(data);
+  setAvatar(data);
+}
+
+function setInfo(data) {
+  name.textContent = data.name;
+  about.textContent = data.about;
+}
+
+function setAvatar(data) {
+  avatar.src = data.avatar;
+}
+
+function onInfoEdit() {
+  setInputValues(profileForm, profileInputList, {
+    name: name.textContent,
+    about: about.textContent,
+  });
+  openModal(profileModal);
+}
+
+function onAvatarEdit() {
+  openModal(avatarModal);
+}
+
+editBtn.addEventListener("click", onInfoEdit);
+avatarBtn.addEventListener("click", onAvatarEdit);
+
+api.getUser().then(setData);
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**